### PR TITLE
refactor vxlan related tests

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/QinQFlowSpec.groovy [hunk 1]
```diff
@@ -842,7 +842,9 @@ class QinQFlowSpec extends HealthCheckSpecification {
         assumeTrue("Unable to find required switches in topology", (allTraffGenSwitches.size() > 1))
         def swP = topologyHelper.getAllNeighboringSwitchPairs().find {
             [it.src, it.dst].every { sw ->
-                sw.dpId in allTraffGenSwitches*.dpId && northbound.getSwitchProperties(sw.dpId).multiTable
+                sw.dpId in allTraffGenSwitches*.dpId && northbound.getSwitchProperties(sw.dpId).multiTable &&
+                        northbound.getSwitchProperties(sw.dpId).supportedTransitEncapsulation
+                                .contains(FlowEncapsulationType.VXLAN.toString().toLowerCase())
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/QinQFlowSpec.groovy:847]: not sure why it's required here. I don't see any vxlan context in this test

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowV2Spec.groovy [hunk 2]
```diff
@@ -241,14 +242,18 @@ class VxlanFlowV2Spec extends HealthCheckSpecification {
...
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find { swP ->
+            [swP.src, swP.dst].every { sw -> isVxlanEnabled(sw.dpId) } && swP.paths.unique(false) {
+                a, b -> a.intersect(b) == [] ? 1 : 0
+            }.size() >= 2
         } ?: assumeTrue("No suiting switches found", false)
 
         def availablePaths = switchPair.paths.findAll { path ->
-            pathHelper.getInvolvedSwitches(path).every { it.noviflow && !it.wb5164 }
+            pathHelper.getInvolvedSwitches(path).every {
+                northbound.getSwitchProperties(it.dpId).supportedTransitEncapsulation
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowV2Spec.groovy:254]: isVxlanEnabled ?

**Review by rtretyak — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/QinQFlowSpec.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/QinQFlowSpec.groovy]: looks like my mistake. fixed

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowV2Spec.groovy (hunk 2 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowV2Spec.groovy]: fixed

**Pull request merged**